resources/assets.py

```diff
@@ -12,6 +12,7 @@
 TAG_SHARP = 'tfc:tools/sharp'
 
 STICKS_WHEN_NOT_SHEARED = loot_tables.alternatives({
+    'name': 'minecraft:stick',
```

Comment:
Typo?

Reply:
Huh, this is unintentional, but also I saw this and deleted it so I don't know why it's back, or how it got there in the first place.